core/domain/story_services.py

```diff
@@ -971,30 +971,31 @@ def record_completed_node_in_story_context(
 
 
 def get_chapter_notification_dicts() -> Tuple[
```

Comment:
This name is now wrong. It should match what's returned.

Reply:
Done